package.json

```diff
@@ -20,7 +20,10 @@
     "test:coverage": "npm run exec-tests",
     "test": "mocha --recursive",
     "lint": "eslint .",
-    "docs": "node scripts/generate-rule-docs.js"
+    "generate-rulesets": "node scripts/generate-rulesets.js && prettier --write conf/rulesets",
+    "docs": "node scripts/generate-rule-docs.js",
+    "prepare": "husky install",
```

Comment:
Sorry if it is a dumb question, but
are the 'prepare' and 'preoublishOnly' needed ?

Reply:
It's not strictly necessary, but it makes docs and ruleset JSON automatically updated when you commit or publish

Reply:
Oh, I see what you're asking.

Those are [`npm` lifecycle scripts](https://docs.npmjs.com/cli/v9/using-npm/scripts#life-cycle-scripts). `npm` will run `prepare` when you run `npm install`, that makes sure that `git` is synced with `husky`.

`npm` will run `prepublishOnly` when you run `npm publish`, that makes sure that `npm` package has up to date rulesets.